app/views/fagus/categories/index.json.jbuilder

```diff
@@ -0,0 +1,4 @@
+json.array! @categories do |category|
+  json.id category.id
+  json.name category.name
```

Comment:
I'm not at the moment, I just left this is as it was a copy and paste and I didn't want to have to look it up as I would probably forget the jbuilder syntax by the time its needed :smile: